File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/javascript/cwe-94/code_before1.js
// Source: Row 2 in ./dataset/CVEfixes/Analysis/results/JavaScript/df_javascript_cwe_94.xlsx

module.exports = function (src, file) {
    if (typeof src !== 'string') src = String(src);
    
    try {
        Function(src);
        return;
    }
    catch (err) {
        if (err.constructor.name !== 'SyntaxError') throw err;
        return errorInfo(src, file);
    }
};

File: experiments_dataset_vul_detect_cwe_classification/dataset/vul_dataset/python/cwe-434/code_before1.py
# Source: Row 2 in ./dataset/CVEfixes/Analysis/results/Python/df_python_cwe_434.xlsx

class ImportFindingsForm(forms.Form):
    class Meta:
        fields = ['engine', 'min_level', 'file']

    engine = forms.CharField(widget=forms.Select(
        attrs={'class': 'form-control form-control-sm'},
        choices=ENGINE_TYPES))
    min_level = forms.CharField(widget=forms.Select(
        attrs={'class': 'form-control form-control-sm'},
        choices=FINDING_SEVERITIES),
        label='Minimum severity')
    file = forms.FileField()
